app/models/ingredient_mention.rb

```diff
@@ -0,0 +1,2 @@
+class IngredientMention < ApplicationRecord
```

Comment:
If you are explicitly defining a has_many for this on ingredients and movies, you need both belongs_to in here